### PR TITLE
New feature

### PR DESCRIPTION
**Pull request by jcodt**
Hi,

A just add new method to get the current executions times, i add exemple in the Readme.md

If you want, you can merge ;-)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ build/logs
 build/pdepend
 cache.properties
 phpunit.xml
+.idea/
```
Comment (GrahamCampbell) [inline .gitignore]: :-1:

File: src/Timer.php [hunk 2]
```diff
@@ -53,6 +53,19 @@ public static function stop()
     }
 
     /**
+     * get current timer and returns the elapsed time.
+     *
+     * @return float
+     */
+    public static function currentTime()
+    {
+        if(!is_array(self::$startTimes) || !array_key_exists(0,self::$startTimes))
```
Comment (GrahamCampbell) [inline src/Timer.php]: cs

**Comment by jcodt** [comment]
Cs ok now

File: README.md [hunk 3]
```diff
@@ -36,6 +34,43 @@ The code above yields the output below:
...
+	
+	PHP_Timer::start();
+
+	while(1 == 1){
+		if(PHP_Timer::currentTime() > 5)
+			throw new Exception("Too long times");
+	}
+
+
+}catch(Exception $e){
```
Comment (GrahamCampbell) [inline README.md]: cs

File: README.md [hunk 4]
```diff
@@ -36,6 +34,43 @@ The code above yields the output below:
...
+
+// ...
+PHP_Timer::currentTime();
+
+```
+
+### Exemple throw Exception after 5s
+```php
+try{
+
```
Comment (GrahamCampbell) [inline README.md]: no space

File: README.md [hunk 5]
```diff
@@ -36,6 +34,43 @@ The code above yields the output below:
...
+PHP_Timer::start();
+
+// ...
+PHP_Timer::currentTime();
+
+```
+
+### Exemple throw Exception after 5s
+```php
+try{
```
Comment (GrahamCampbell) [inline README.md]: missing space

**Comment by GrahamCampbell** [comment]
> Cs ok now

It seems to have got worse?

File: src/Timer.php [hunk 6]
```diff
@@ -53,6 +53,20 @@ public static function stop()
     }
 
     /**
+     * get current timer and returns the elapsed time.
+     * @return float
+     * @throws \Exception
+     */
+    public static function currentTime()
+    {
+        if(!is_array(self::$startTimes) || !array_key_exists(0,self::$startTimes)) {
```
Comment (GrahamCampbell) [inline src/Timer.php]: cs

File: src/Timer.php [hunk 7]
```diff
@@ -53,6 +53,20 @@ public static function stop()
     }
 
     /**
+     * get current timer and returns the elapsed time.
```
Comment (GrahamCampbell) [inline src/Timer.php]: need a capital G and a new line after the description.

Comment (GrahamCampbell) [inline src/Timer.php]: Also, this description is wrong. "Get the elapsed time." would be a better description. The description doesn't need to explain the internal.

**Comment by jcodt** [comment]
ok

**Comment by sebastianbergmann** [comment]
Thank you for your contribution. However, I am not interested in adding new features to this component.

**Pull request closed without merging**